dbus/dbus.c

```diff
@@ -31,7 +31,7 @@ bool init_dbus(struct mako_state *state) {
 
 	ret = sd_bus_request_name(state->bus, service_name, 0);
 	if (ret < 0) {
-		fprintf(stderr, "Failed to acquire service name: %s\n", strerror(-ret));
+		fprintf(stderr, "Failed to acquire service name: %s. Is Mako already running?\n", strerror(-ret));
```

Comment:
Maybe we can check the value of `ret` and only print the new message in case it's already running?